poc/tests/test_vidpf.py

```diff
@@ -247,7 +247,7 @@ def test_malformed_correction_word_ctrl(self):
         malformed_level = randrange(vidpf.BITS)
         (seed_cw, ctrl_cw, w_cw, proof_cw) = public_share[malformed_level]
         malformed = ctrl_cw.copy()
-        malformed[randrange(2)] += Field2(1)
+        malformed[randrange(2)] ^= True
```

Comment:
Can we use the Boolean `not` instead of XOR here?

Reply:
Done.